src/tooltip.js

```diff
@@ -47,8 +47,8 @@ nv.models.tooltip = function() {
      */
     var position = function() {
         return {
-            left: d3.event.clientX,
-            top: d3.event.clientY
+            left: d3.event !== null ? d3.event.clientX : 0,
+            top: d3.event !== null ? d3.event.clientY : 0
```

Comment:
This is needed because when data are updated, if the tooltip is currently display, it can stay visible even if the mouse is not on the point anymore, because no mouse move event is detected. Later if I go out of the chart the tooltip is still displaying (and never hide until going on another point). To avoid that I called chart.tooltip.hidden(true) at each update, but d3.event is null in this case.